src/components/Menu/Menu.component.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useContext } from 'react';
+import styled from 'styled-components';
+import { useHistory } from 'react-router-dom';
+
+import { StateContext } from '../../context/State/state';
+
+const StyledMenu = styled.nav`
```

Comment:
You can move the styled components to another file `Menu.styled.js` and just import all the styles here